go.mod

```diff
@@ -202,7 +202,7 @@ require (
 	github.com/pion/interceptor v0.1.12 // indirect
 	github.com/pion/logging v0.2.2 // indirect
 	github.com/pion/mdns v0.0.7 // indirect
-	github.com/pion/mediadevices v0.4.1-0.20230320003250-11bf55f80ca8 // indirect
```

Comment:
why is this changing?

Reply:
this appears to have bumped to match the version in rdk https://github.com/viamrobotics/rdk/blob/main/go.mod#L65